src/app/community-view/community-view.component.ts

```diff
@@ -123,4 +129,20 @@ export class CommunityViewComponent implements OnInit {
     // Dirty fix because of: https://github.com/angular/angular/issues/17772
     this.changeDetector.markForCheck();
   }
+
+  private showInfoDialog(community: CommunityResponse) {
+    this.dialog.open(ClosedCommunityInfoDialogComponent, {
```

Comment:
Can we just pass the `community` object into `ClosedCommunityInfoDialogComponent` without copying it?